packages/plugin-core/src/components/lookup/utils.ts

```diff
@@ -437,6 +438,15 @@ export class PickerUtilsV2 {
     }
   }
 
+  static isCreateNewNoteWithTemplatePick(
```

Comment:
should this be `Picked`?

Reply:
went with `Pick` since the counterpart is named `isCreateNewNotePick` but I can fix both.